### DISCUSSION TITLE
Events - Time Filter

### DISCUSSION THREAD

**Discussion by idle-user** [blakeblackshear/frigate#6685] [Ideas]
Has the idea of filtering events from time, and not just date been brought up?

I have multiple cameras and events, and sometimes I want to see events from only a specific hour. Scrolling down for a while doesn't seem very optimal.

Thanks,

**Comment by NickM-27** [comment]
There is a feature request and a PR implementing it that is up

**Reply by idle-user** [reply]
fantastic. Thank you
